.rubocop.yml

```diff
@@ -0,0 +1,7 @@
+# Allow long lines.
```

Comment:
Since this project is not primarily a Ruby project, I'd rather avoid adding Ruby-specific tooling/config for it; can you take this file out of the PR?